tests/queries/tests.py

```diff
@@ -3677,7 +4024,10 @@ def test_ticket_23605(self):
 
 
 class TestTicket24279(TestCase):
-    def test_ticket_24278(self):
+    def test_q_objects_with_empty_lists(self):
+        """
+        squashmigrations throws ValueError: need more than 1 value to unpack (#24278)
```

Comment:
There's a typo 24278 should be 24279 -- if you put some into it, you'd see this has nothing to do with squashmigrations.